src/etc/inc/filter.lib.inc

```diff
@@ -90,13 +90,23 @@ function filter_core_get_antilockout()
 {
     global $config;
 
+    // Migration from old noantilockout option
     if (isset($config['system']['webgui']['noantilockout'])) {
-        return [];
+      unset($config['system']['webgui']['noantilockout']);
+    }
+    if (!isset($config['system']['webgui']['antilockout_interface'])) {
+      $lockout_if = get_primary_interface_from_list(array_keys(legacy_config_get_interfaces(['virtual' => false, 'enabled' => true])));
```

Comment:
User view: Why are we adding compatibility but don't provide a way to turn this off anymore? I just want the old setting to remain and work as before.